cli/bpmetadata/proto/bpmetadata.proto

```diff
@@ -0,0 +1,332 @@
+syntax = "proto3";
+
+package google.cloud.config.bpmetadata;
+
+option go_package = "github.com/GoogleCloudPlatform/cloud-foundation-toolkit/cli/bpmetadata";
+
+import "bpmetadata_ui.proto";
+
+message BlueprintMetadataSpec {
```

Comment:
I don't think we need to instantiate them because they are intended to be anonymous fields. WIth protos you define them as messages without instantiation and `protoc` for Go creates types like `BlueprintMetadataSpec_BlueprintInfo` etc for access their fields. In other words, we don't need any fields inside the `BlueprintMetadataSpec` struct because it's just placeholder for `BlueprintInfo`, `BlueprintContent` etc.

The alternative would've been to flatten everything under `BlueprintMetadataSpec` and lose the anonymous fields. That seemed messier to me.

Reply:
Without defining fields in `BlueprintMetadataSpec` how would we assign the value to an instance of `BlueprintMetadataSpec`?  For example wouldn't something like
```
BlueprintMetadataSpec_builder{
		title : "foo",
	}.Build()
```
be invalid because `BlueprintMetadataSpec` does not have a field called `title` even if `BlueprintInfo` has it?

Reply:
Instantiated fields as per our discussion.